### ISSUE TITLE
Unexpected token function for `async`

### ISSUE THREAD

**Issue by bzang** [wbyoung/babel-plugin-transform-postcss#33]
When I try using the transform I get the following when running with Node 6:
```
/node_modules/babel-plugin-transform-postcss/dist/postcss-server.js:56
const main = async function main(socketPath, tmpPath) {
                   ^^^^^^^^
SyntaxError: Unexpected token function
```

I don't have any issues when running with Node 7, but our current architecture doesn't quite support 7 yet. Any thoughts on how I can get around this?

My `.babelrc`:
```
{
  "plugins": [
    ["babel-plugin-transform-builtin-extend", {
        "globals": ["Error"]
    }],
    ["transform-runtime", {
      "polyfill": false
    }],
    "transform-decorators-legacy",
    "transform-export-extensions",
    "transform-object-rest-spread",
    "lodash"
  ],
  "presets": [
    "es2015",
    ["env", {
      "targets": {
        "node": "current"
      }
    }]
  ]
}
```

**Comment by JaKXz** [comment]
I am in the same boat and would have thought that `babel-preset-env` set to `"current"` would fix the problem. Not sure why it isn't though. :(

**Comment by wbyoung** [comment]
[This](https://github.com/wbyoung/babel-plugin-transform-postcss/blob/d8faa1dcdf2b1ea51ce997b5ab37e6f62e576992/.babelrc#L5) just needs to change to be a specific version and we need to update [thetested versions](https://github.com/wbyoung/babel-plugin-transform-postcss/blob/d8faa1dcdf2b1ea51ce997b5ab37e6f62e576992/.travis.yml#L3).

Would you mind creating a PR and testing with your configuration?

**Comment by wbyoung** [comment]
Resolved by #38.
